applications/matter_weather_station/src/app_task.cpp

```diff
@@ -120,19 +125,18 @@ void AppTask::OpenPairingWindow()
 {
 	/* Don't allow on starting Matter service BLE advertising after Thread provisioning. */
 	if (ConnectivityMgr().IsThreadProvisioned()) {
-		LOG_INF("NFC Tag emulation and Matter service BLE advertisement not started - device is commissioned to a Thread network.");
+		LOG_INF("NFC Tag emulation and Matter service BLE advertising not started - device is commissioned to a Thread network.");
 		return;
 	}
 
 	if (ConnectivityMgr().IsBLEAdvertisingEnabled()) {
-		LOG_INF("BLE Advertisement is already enabled");
+		LOG_INF("BLE advertising is already enabled");
 		return;
 	}
 
-	if (OpenDefaultPairingWindow(chip::ResetFabrics::kNo) == CHIP_NO_ERROR) {
-		LOG_INF("Enabled BLE Advertisement");
```

Comment:
Don't you think that log is useful? I mean it seems to be a good user experience if you want to do something by pressing a button and get a log confirming it went fine.

Reply:
The reason I removed this log is that the device layer adds two more log messages about starting BLE advertising, so thought another one was useless :)

Reply:
Oh ok, nevermind then